test/validation/api/queue/queue.c

```diff
@@ -461,7 +462,7 @@ static void queue_test_info(void)
 	CU_ASSERT(info.param.sched.sync == odp_queue_sched_type(q_order));
 	CU_ASSERT(info.param.sched.group == odp_queue_sched_group(q_order));
 	ret = odp_queue_lock_count(q_order);
-	CU_ASSERT(ret > 0);
+	CU_ASSERT(ret == param.sched.lock_count);
```

Comment:
Won't this `CU_ASSERT()` fail if `odp_queue_lock_count()  > 1` since line 435 sets `param.sched.lock_count = 1`?

Reply:
True, will fix.

Reply:
On second thought, this should work since odp_queue_lock_count() should return the lock_count used in odp_queue_create() parameters. So, a case you described should actually fail.

Reply:
Agreed. Test is good as currently stated, however there's considerable overlap with the similar `CU_ASSERT()` on line 466. Perhaps just combine them? E.g., 
```
CU_ASSERT(odp_queue_lock_count(q_order) == info.param.sched.lock_count);
```

Reply:
This would decrease code coverage a bit since both `odp_queue_info()` and `odp_queue_lock_count()` could potentially return invalid values. `param.sched.lock_count`is set by the test, so it's always safe to compare to.

Reply:
Good point.